go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/metal3-io/baremetal-operator
 
 go 1.23.7
 
+toolchain go1.24.1
```

Comment:
This is dependabot issue, which is being fixed.

/hold
until it is and this can be recreated right, or manually fixed.

Reply:
It is now fixed.
/unhold